shortest.config.ts

```diff
@@ -3,7 +3,7 @@ import type { ShortestConfig } from "@antiwork/shortest";
 export default {
   headless: false,
   baseUrl: "http://localhost:3000",
-  testDir: "app/__tests__",
+  testDir: ["app/__tests__", "examples"],
```

Comment:
Added examples test to be included in test dir. This will run in the CI as well.